### PR TITLE
{tools}[dummy/] NcFTP v3.2.6

### PR DESCRIPTION
**Pull request by pescobar**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/n/NcFTP/NcFTP-3.2.6.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+version = '3.2.6'
+
+homepage = 'http://www.ncftp.com/ncftp/'
+description = """NcFTP Client (also known as just NcFTP) is a set of FREE application programs
+ implementing the File Transfer Protocol (FTP)"""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['ftp://ftp.ncftp.com/ncftp/']
+sources = ['%(namelower)s-%(version)s-src.tar.gz']
```
Comment (verdurin) [inline easybuild/easyconfigs/n/NcFTP/NcFTP-3.2.6.eb]: Please add a checksum.

File: easybuild/easyconfigs/n/NcFTP/NcFTP-3.2.6.eb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+easyblock = 'ConfigureMake'
+
+name = 'NcFTP'
+version = '3.2.6'
+
+homepage = 'http://www.ncftp.com/ncftp/'
+description = """NcFTP Client (also known as just NcFTP) is a set of FREE application programs
+ implementing the File Transfer Protocol (FTP)"""
+
+toolchain = {'name': 'dummy', 'version': ''}
```
Comment (verdurin) [inline easybuild/easyconfigs/n/NcFTP/NcFTP-3.2.6.eb]: Any specific reason why this is `dummy`?

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 1 year.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Comment by ghuls** [comment]
```
cat NcFTP-3.2.6-foss-2018a.eb
# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
# Author: Pablo Escobar Lopez
# sciCORE - University of Basel
# SIB Swiss Institute of Bioinformatics 

easyblock = 'ConfigureMake'

name = 'NcFTP'
version = '3.2.6'

homepage = 'https://ncftp.com/ncftp/'
description = """NcFTP Client (also known as just NcFTP) is a set of FREE application programs
 implementing the File Transfer Protocol (FTP)"""

toolchain = {'name': 'foss', 'version': '2018a'}

source_urls = ['ftp://ftp.ncftp.com/ncftp/']
sources = ['%(namelower)s-%(version)s-src.tar.gz']
checksums = ['129e5954850290da98af012559e6743de193de0012e972ff939df9b604f81c23']

sanity_check_paths = {
    'files': ["bin/%s" % x for x in ["ncftp", "ncftpbatch", "ncftpget",
                                     "ncftpls", "ncftpput", "ncftpspooler"]],
    'dirs': [],
}

moduleclass = 'tools'
```

**Pull request closed without merging**